models/party.js

```diff
@@ -21,8 +21,15 @@ class Party {
 
     const newParty = new Parse.Object("Party");
     newParty.set("name", body.name)
-    const query = new Parse.Query("User");
-    const dm = await query.get(body.dm.objectId);
+    var dm
```

Comment:
using `var` in Javascript is frowned upon because it's suppose to be module scoped.  Use `let` instead if you need to mutate the variable.